nibabel/parrec.py

```diff
@@ -609,7 +609,7 @@ def __getitem__(self, slicer):
                                  'F')
         # Broadcast scaling to shape of original data
         slopes, inters = self._slice_scaling
-        fake_data = strided_scalar(self._shape)
```

Comment:
This works:
`_, slopes, inters = np.broadcast_arrays(fake_data.copy(), slopes.copy(),
inters.copy())`

Just gotta narrow down from there.


On Wed, Oct 14, 2015 at 7:30 PM, Chris Markiewicz <notifications@github.com>
wrote:

> In nibabel/parrec.py
> <https://github.com/nipy/nibabel/pull/358#discussion_r42077733>:
>
> > @@ -609,7 +609,7 @@ def __getitem__(self, slicer):
> >                                   'F')
> >          # Broadcast scaling to shape of original data
> >          slopes, inters = self._slice_scaling
> > -        fake_data = strided_scalar(self._shape)
>
> Just tried. self._shape is a tuple, so that doesn't work. (Failed to try
> the right tests, locally. Want test_proxy_api, if you're playing along at
> home.)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nipy/nibabel/pull/358/files#r42077733>.
>